server.go

```diff
@@ -243,6 +245,42 @@ func (s *Server) register(sd *ServiceDesc, ss interface{}) {
 	s.m[sd.ServiceName] = srv
 }
 
+// Metadata returns the metadata for a given symbol name.
+// The name can be a service name or a method name in the form of
+// <package>.<service>[.<method>].
+func (s *Server) Metadata(name string) interface{} {
```

Comment:
Calling it Metadata is misleading because it does NOT return the metadata of a grpc server. Instead it returns the metadata of a service registered on the server.